assets/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "env": {
     "browser": true
+  },
+  "rules": {
+    "strict": "off"
```

Comment:
`"node": false` would mean I need to add `/* global: require */` as well - but okay, I'll get them :sweat_smile: